src/simulation/dynamics/gravityEffector/polyhedralGravityModel.rst

```diff
@@ -66,6 +66,7 @@ For gravity evaluations, the polyhedron gravity has to be initialized
     erosGravBody.initializeParameters() # Initializes polyhedron module internal variables
     pos = [40*1e3, 30*1e3, 10*1e3] # Evaluation point
     acc = erosGravBody.computeField(pos) # Gravity evaluation
-    
+
 Supported polyhedron shape files
-----------
+--------------------------------
+text
```

Comment:
Either add content to this section, or delete the section title.
I fixed the underscore length to match the title text length already.

Reply:
Done, thank you. All tests pass in commit b87f027eed7c259d95a2d8d80316db7e4a2b4346

I've been trying to squash my last 3 commits but weird merges are happening